ui/StatusQ/src/StatusQ/Controls/StatusTagItem.qml

```diff
@@ -68,49 +68,52 @@ Rectangle {
 
     width: tagRow.implicitWidth + 2 * d.tagMargins
```

Comment:
You don't need this at all, the `Control` will take care of default sizing

Reply:
what do you mean, this is the main sizing of the component, not the contentItem